hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/execution/benchmark/LSMTimelineReadBenchmark.scala

```diff
@@ -42,8 +42,9 @@ object LSMTimelineReadBenchmark extends HoodieBenchmarkBase {
    * Apple M2
    * pref load archived instants:              Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
    * ------------------------------------------------------------------------------------------------------------------------
-   * read shim instants                                   18             32          15          0.1       17914.8       1.0X
-   * read instants with commit metadata                   19             25           5          0.1       19403.1       0.9X
+   * read slim instants                                  494            521          27          0.5        1899.6       1.0X
+   * read instants with commit metadata                 2544           2625         116          0.1        9785.9       0.2X
+   * read start time                                     156            177          26          1.7         601.1       3.2X
```

Comment:
Just for my understanding, why is reading timeline with start time 3.2x slower? I did expect it to be a little slower but 3.2x sound a big difference.

Reply:
The archived timeline instants loading firstly buffer all the instants in memory then sort all the instants by start time, that sort takes time.

Reply:
Got it and this was done with how many instants in archived timeline?

Reply:
260000, 3 months commits with 30s as interval.